test/integration_test/stork-specs/prometheus/alert_manager/alert_manager.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: stork #This name is important since the Alertmanager pods won't start unless a secret named alertmanager-${ALERTMANAGER_NAME} is created. in this case, if would expect alertmanager-stork secret in the kube-system namespace
+  namespace: kube-system
+  labels:
+    alertmanager: stork
+spec:
+  replicas: 3
```

Comment:
nit: missing new line..there are some other files in this PR that are missing a new line at the end